client/state/sites/reducer.js

```diff
@@ -2,12 +2,22 @@
  * External dependencies
  */
 import { combineReducers } from 'redux';
+import pick from 'lodash/pick';
 
 /**
  * Internal dependencies
  */
 import { plans } from './plans/reducer';
-import { SITE_RECEIVE, SERIALIZE, DESERIALIZE } from 'state/action-types';
+import { SITE_RECEIVE, DESERIALIZE } from 'state/action-types';
+import { sitesSchema } from './schema';
+import { isValidStateWithSchema } from 'state/utils';
+
+/**
+ * Module variables
+ */
+// [TODO]: This validation is only necessary so long as we continue to receive
+// decorated sites from the `lib/sites-list` module.
+const validSiteKeys = Object.keys( sitesSchema.patternProperties['^\\d+$'].properties );
```

Comment:
Looking at the rest endpoint, it looks like I missed two entries. Do you mind adding this to `sites/schema.js`?
```php
'updates'           => '(array) An array of available updates for plugins, themes, wordpress, and languages.',
'jetpack_modules'   => '(array) A list of active Jetpack modules.',
```